Clean up new documentation (#10174)
streetwriters/notesnook: docs/help/contents/plans-and-limits.md

## Education plan

> The Education plan gives students and teachers **Pro features and Pro limits**. It is not sold from the plan grid. [Contact us](https://notesnook.com/education) to apply. Education plans do not include a trial.

Reviewer: ```suggestion
The Education plan gives students and teachers **Pro features and Pro limits**. It is not sold from the plan grid. You can [apply for the discount here](https://notesnook.com/education). Education plans do not include a trial.
```
